kqueen/engines/resources/heat/default/fragments/docker.service.yaml

```diff
@@ -0,0 +1,58 @@
+#cloud-config
+merge_how: dict(recurse_array)+list(append)
+bootcmd:
+  - mkdir -p /etc/systemd/system/docker.service.d
+  - mkdir -p /etc/systemd/system/flanneld.service.d
+write_files:
+  - path: /usr/local/bin/flannel-docker-bridge
```

Comment:
does it possible to use calico?
afaik in future we need to support calico-managing and network range specifying